### ISSUE TITLE
Logo on Invoice disappeared without any changes being made 1.7.8.5

### ISSUE THREAD

**Issue by skellatore** [PrestaShop/PrestaShop#28207]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Literally as the title states, without any modification to code, templates or the BO (save for accepting/processing orders and add/edit/remove products), Invoices - past and present, No longer have the logo displayed. This is also happening on my dev site which hasn't been logged into for months and is on the version 1.7.8.3.

No errors in any logs, no errors in the logs of Prestashop.

I've tried upgrading to the latest version (I was on 1.7.8.3, now on 1.7.8.5) - No Change
Upgraded the PHP Version - No Change
Re-Uploaded the Logo for the Main, Mail and Invoice sections - No Change.
Cleared Cache - No Change
Disabled Cache - No Change
Checked SSL Certificates - All Correct and NOT self-signed.
Checked the image file exists and is public accessible - It is
Contacted Hosting Service provider to see if any maintenance or patches has taken place - Nothing has changed.

I literally have no explanation as to why this has started happening and how I can even go about fixing this problem

### Expected behavior

I'd expect there to be an error at the very least so I can fix... or better yet the Logo just to work like it was.

### Steps to reproduce

Not sure how to reproduce because nothing had changed!

### PrestaShop version(s) where the bug happened

1.7.8.5

### PHP version(s) where the bug happened

7.4.28

**Comment by skellatore** [comment]
I was able to get debug mode activated and I have received this error when trying to open an Invoice:
TCPDF ERROR: GD library error: Warning: imagepng(~/public_html/tmp/__tcpdf_dcfaba37bd81403056beac39af62805e_imgmask_alpha_91ef3cfacdc8bc99884f86baae2918a5): failed to open stream: No such file or directory

I would truely love to know where the it got ~/public_html/tmp from because that's not at all where the temporary files should be stored, and that I cant find that definition anywhere.

**Comment by skellatore** [comment]
I've been able to track down and fix the problem.. To help someone else in future, my fault some how managed to be related to a PHP variable "upload_tmp_dir".

It was set to ~/public_html/tmp
I changed it to an absolute address /home/<myhostingaccountusername>/tmp 

Immediately this was resolved. I've logged a ticket to my Hosting service provider and will also ask them why they weren't completely telling the truth about not changing anything.
